conf/janus.plugin.streaming.jcfg.sample.in

```diff
@@ -249,4 +249,31 @@ file-ondemand-sample: {
 	#rtsp_user = "username"
 	#rtsp_pwd = "password"
 	#secret = "adminpwd"
+
```

Comment:
This whole new section doesn't belong here, but on top, where the other RTSP properties are introduced (look for `# The following options are only valid for the 'rstp' type:`). The same should then be added to the related section in the code, as that's what generates the documentation.

Reply:
Thank you very much for your review!